localstripe/localstripe-v3.js

```diff
@@ -151,17 +164,25 @@ Stripe = (apiKey) => {
   return {
     elements: () => {
       return {
-        create: (type, options) => {
+        create: function(type, options) {
           console.log('localstripe: Stripe().elements().create()');
-          return new Element();
+          if (this.cardElement) {
+            throw new Error("Can only create one Element of type card");
+          }
+          this.cardElement = new Element(this);
+          return this.cardElement;
         },
+        getElement: function(type) {
+          console.log('localstripe: Stripe().elements().getElement()');
```

Comment:
Is this log line very valuable? If not, I'd prefer not spamming the console and remove it.

Reply:
It doesn't matter to me. I was just copying what I saw elsewhere, so I'll take it out.